app/src/NativeBase/Containers/ProfileContainer.tsx

```diff
@@ -13,20 +44,72 @@ import React from 'react'
 const ProfileContainer = () => {
   return (
     <VStack
-      paddingTop="4"
-      paddingBottom="2"
-      alignItems="center"
-      space="4"
-      backgroundColor="secondaryGrey.60"
+      padding={2}
+      backgroundColor="bg.100"
       _dark={{ backgroundColor: 'bgDarkMode.100' }}
     >
-      <Center
-        backgroundColor="bg.100"
-        _dark={{ backgroundColor: 'bgDarkMode.100' }}
-        width="100%"
-      >
-        <Heading size="sm">My Profile</Heading>
+      <Center>
+        <Heading fontFamily="title" size="lg">
+          My Profile
+        </Heading>
+        <Box
+          marginTop={6}
+          height="93px"
```

Comment:
Is there a reason the width and height are different values, if it's a circle?

Reply:
I've followed the design here, thinking it is intentionally not actually a circle? Happy to change though. Might need to ask Mia?